classes/Task/Update/Unzip.php

```diff
@@ -89,9 +93,9 @@ public function run(): int
         // From PrestaShop 1.7, we zip all the files in another package
         // which must be unzipped too
         $newZip = $destExtract . DIRECTORY_SEPARATOR . 'prestashop.zip';
-        if (file_exists($newZip)) {
-            @unlink($destExtract . DIRECTORY_SEPARATOR . '/index.php');
-            @unlink($destExtract . DIRECTORY_SEPARATOR . '/Install_PrestaShop.html');
+        if ($this->container->getFileSystem()->exists($newZip)) {
+            $this->container->getFileSystem()->remove($destExtract . DIRECTORY_SEPARATOR . '/index.php');
+            $this->container->getFileSystem()->remove($destExtract . DIRECTORY_SEPARATOR . '/Install_PrestaShop.html');
```

Comment:
```suggestion
            $this->container->getFileSystem()->remove([
                $destExtract . DIRECTORY_SEPARATOR . '/index.php'),
                $destExtract . DIRECTORY_SEPARATOR . '/Install_PrestaShop.html',
            ]);
```

The result is the same but it shows we can call it only one time.